apps/webapp/app/routes/_app.orgs.$organizationSlug.v3.usage/route.tsx

```diff
@@ -207,14 +207,12 @@ export default function Page() {
                     <>
                       <Table>
                         <TableHeader>
-                          <TableRow>
-                            <TableHeaderCell>Task</TableHeaderCell>
-                            <TableHeaderCell alignment="right">Runs</TableHeaderCell>
-                            <TableHeaderCell alignment="right">Average duration</TableHeaderCell>
-                            <TableHeaderCell alignment="right">Average cost</TableHeaderCell>
-                            <TableHeaderCell alignment="right">Total duration</TableHeaderCell>
-                            <TableHeaderCell alignment="right">Total cost</TableHeaderCell>
-                          </TableRow>
+                          <TableHeaderCell>Task</TableHeaderCell>
+                          <TableHeaderCell alignment="right">Runs</TableHeaderCell>
+                          <TableHeaderCell alignment="right">Average duration</TableHeaderCell>
+                          <TableHeaderCell alignment="right">Average cost</TableHeaderCell>
+                          <TableHeaderCell alignment="right">Total duration</TableHeaderCell>
+                          <TableHeaderCell alignment="right">Total cost</TableHeaderCell>
```

Comment:
"You must place `<tr>` elements inside `<thead>` to contain your `<th>` cells. The correct structure is: `<thead><tr><th>Header Cell</th></tr></thead>`. [1](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead) [2](https://www.w3.org/TR/html401/struct/tables.html)"